rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -161,6 +166,7 @@ def create_inquirer(
         should_mock_current_price_queries,
         mocked_prices,
         current_price_oracles_order,
+        ignore_mocked_prices_for=(),
```

Comment:
Should it be an empty tuple or an empty list. The types here are off. And as you know empty list as default argument won't work.

Just use `None` as default argument and below compare is not None